libraries/ssh_known_hosts.rb

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env ruby
+
+module SshKnownHosts
```

Comment:
file and module now renamed to `KnownHosts`, seems correct to me
or should be named after the new name of the repo ssh_known_hosts_ng (SshKnownHostsNG)?
@andytson-inviqa